src/components/Thumbnail.scss

```diff
@@ -0,0 +1,97 @@
+@import "../constants.scss";
+
+.thumbnail__wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+}
+
+.thumbnail__left {
+  width: 60%;
+  height: 100%;
+  margin-right: 12px;
+}
+
+.thumbnail__left-inner {
+  display: flex;
+  justify-content: flex-start;
+  width: 100%;
+}
+
+.thumbnail__right {
+  width: calc(100% - 60% - 12px);
+  height: 100%;
+}
+
+.thumbnail__right-inner {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  width: 100%;
+}
+
+.thumbnail__picture {
+  width: 100%;
+  width: 80px;
+  height: 100%;
+  height: 60px;
+  margin-right: 12px;
+  border-radius: 8px;
+  background: url(./icons/thumbnail.svg) center no-repeat;
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.thumbnail__img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.thumbnail__item-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: calc(100% - 94px);
+  height: 100%;
+}
+
+.thumbnail__item-name,
+.thumbnail__item-size {
+  margin: 0 0 5px 0;
+}
+
+.thumbnail__item-name {
+  font-family: $font;
+  font-style: normal;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 150%;
+  color: #1B1B1F;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.thumbnail__item-size {
+  font-family: $font;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 150%;
+  color: #D1D1D1;
+}
+
+.svg__change {
+  width: 20px;
+  height: 20px;
+  background: url(./icons/changeFile.svg) no-repeat center;
+  background-size: 15px 17px;
+  margin-right: 6px;
+}
+
+.button__notsee {
+  display: none;
+}
```

Comment:
Please, add empty line at the end of css, ts, tsx, js, jsx files
With an empty line it is easear to add new code

Reply:
done